stable/atlantis/values.yaml

```diff
@@ -81,6 +81,9 @@ image:
 ## enable using atlantis.yaml file
 allowRepoConfig: false
 
+## enable apply requirements flag
```

Comment:
Again, this should be clear that it's only talking about the approval requirement.

```suggestion
# Require all pull requests be approved prior to apply. 
```